android/app/src/main/java/app/organicmaps/downloader/BottomPanel.java

```diff
@@ -135,23 +117,15 @@ public void update()
         {
           switch (status)
           {
-          case CountryItem.STATUS_UPDATABLE:
-            UpdateInfo info = MapManager.nativeGetUpdateInfo(root);
-            setUpdateAllState(info);
-            break;
-
-          case CountryItem.STATUS_DONE:
-            show = false;
-            break;
-
-          case CountryItem.STATUS_PROGRESS:
-          case CountryItem.STATUS_APPLYING:
-          case CountryItem.STATUS_ENQUEUED:
-            setCancelState();
-            break;
-
-          default:
-            setDownloadAllState();
+            case CountryItem.STATUS_UPDATABLE ->
+            {
+              UpdateInfo info = MapManager.nativeGetUpdateInfo(root);
+              setUpdateAllState(info);
+            }
+            case CountryItem.STATUS_DONE -> show = false;
+            case CountryItem.STATUS_PROGRESS, CountryItem.STATUS_APPLYING, CountryItem.STATUS_ENQUEUED ->
+                setCancelState();
```

Comment:
Please review if there are similar changes. Fewer lines of (a simpler) code are preferred, they make reviews easier.

```suggestion
            case CountryItem.STATUS_PROGRESS, CountryItem.STATUS_APPLYING, CountryItem.STATUS_ENQUEUED -> setCancelState();
```